bindings/python/visualize/meshcat_visualizer.py

```diff
@@ -633,7 +633,12 @@ def load_model(self, model: idyn.Model, model_name="iDynTree", color=None):
         self.traversal[model_name] = idyn.Traversal()
         self.link_pos[model_name] = idyn.LinkPositions()
 
-        self.model[model_name].computeFullTreeTraversal(self.traversal[model_name])
+        if base_frame is None:
+            self.model[model_name].computeFullTreeTraversal(self.traversal[model_name])
+        else:
+            base_frame_index = self.model[model_name].getFrameIndex(base_frame)
+            self.model[model_name].computeFullTreeTraversal(self.traversal[model_name], base_frame_index)
```

Comment:
Should we handle somehow the case in which `base_frame` is not a valid name?

Reply:
Done!

Reply:
https://github.com/robotology/idyntree/pull/1229/commits/08b75f21c416fd5960058093d77b23a7b0b2b1eb

Reply:
@GiulioRomualdi I may be missing something, but it does not seem that https://github.com/robotology/idyntree/commit/08b75f21c416fd5960058093d77b23a7b0b2b1eb handles the case in which `base_link` is not a valid name?

Reply:
My fault : https://github.com/robotology/idyntree/pull/1229/commits/d1d688db847cbb417041f58f6a9d9c1a3a890739

Reply:
Actually I got
```
 if base_link_index == idyn.LINK_INVALID_INDEX:
AttributeError: module 'idyntree.bindings' has no attribute 'LINK_INVALID_INDEX'
```

Reply:
Strange, it seems exposed at least on MATLAB: https://github.com/robotology/idyntree/blob/83bd07a89476060301aa83da5de580f82c69fdc9/bindings/matlab/autogenerated/%2BiDynTree/LINK_INVALID_INDEX.m#L4 . Anyhow, you can hardcode `-1`, see https://github.com/robotology/idyntree/blob/83bd07a89476060301aa83da5de580f82c69fdc9/src/model/src/Indices.cpp#L9 .